run-tests.ps1

```diff
@@ -5,12 +5,23 @@ if (-Not $HasTestsDir -Or -Not $HasProjectDir) {
     exit 2
 }
 
+# Credit: https://stackoverflow.com/a/54935264
+function New-TemporaryDirectory {

```

Comment:
Codacy found an issue: [Function 'New-TemporaryDirectory' has verb that could change system state. Therefore, the function has to support 'ShouldProcess'.](https://app.codacy.com/manual/wwkimball/yamlpath/pullRequest?prid=6403599)

Reply:
I added precisely the line of code which Codacy requires, yet it has failed to notice the change.  I am therefore going to force Codacy to IGNORE this issue; it is obviously a BUG in Codacy causing a false positive against my code.